src/components/CombinePopup.vue

```diff
@@ -75,11 +78,54 @@ export default {
     })
     return {
       selected: null,
-      options: options
+      options: options,
+      bandCounts: null,
+      featureCount: 0
     }
   },
-  mounted () {
+  async mounted () {
+    let scenarioIds = this.path.split(',').map(_.toNumber)
+    let scenarioInfo = await this.getScenarioInfo(scenarioIds)
+    /* extract number of features and bands per layer */
+    this.$set(this, 'bandCounts', scenarioInfo.properties.bandCounts)
+    this.$set(this, 'featureCount', scenarioInfo.features.length)
+
+    /* select first option (waterlevel) by default */
     this.selected = this.options[0].id
+  },
+  methods: {
+    getBandCount (option) {
+      return _.get(this.bandCounts, option.name, '')
+    },
+    async getScenarioInfo (scenarioIds) {
+      let services = await mapConfig.getServices()
```

Comment:
a config, not an env? 
thought it a weird config (is that the way to use comments in there?)

Reply:
not sure what you mean here. The mapConfig is loaded from the public folder. Because we deliver a precompiled static zip file, where the config needs to be changed later. So I kept a config file that can be changed apart from the webpack run.